### ISSUE TITLE
Incosistency in python code examples

### ISSUE THREAD

**Issue by 01-vyom** [keras-team/keras#14971]
**URL(s) with the issue:**

https://keras.io/api/models/sequential/#sequential-class

**Description of issue (what needs to be changed):**

There is an inconsistency in the Keras python codes in doc in the Sequential Class. 

<img width="741" alt="Screenshot 2021-07-20 at 6 41 38 PM" src="https://user-images.githubusercontent.com/46242526/126331306-780a5ca8-4c87-4a01-bb58-0bf0d446aef2.png">

This inconsistency lies in several other places where the code hasn't been wrapped with the python keyword.

Some other locations:
https://keras.io/api/layers/activation_layers/softmax/
https://keras.io/api/layers/activation_layers/relu/

It becomes tedious to work with while copying the code.

**Correct links**

https://github.com/keras-team/keras/blob/00518dcf7f5b7de7064267979d7e9f9916e777eb/keras/engine/sequential.py#L41-L96

**Submit a pull request?**
Yes

**Comment by rmothukuru** [comment]
@01-vyom,
Thank you for the issue and appreciate your interest to submit a PR. Please create a PR so that Keras Engineers can review it. Thanks!

**Comment by 01-vyom** [comment]
Ok, I will make a PR with changes to some of these files.

**Comment by rmothukuru** [comment]
@01-vyom,
Can you please let us know what exactly is the inconsistency(sorry for my ignorance) that you are observing so that I can fix it? Thanks!

**Comment by 01-vyom** [comment]
@rmothukuru 
Some of the examples have `>>>` and others don't. That is why it is inconsistent. Also, it becomes difficult to copy the code with `>>>` at the start. I am preparing a PR if that is okay with you.

**Comment by rmothukuru** [comment]
@01-vyom,
Sure. Please go ahead.

**Comment by ymodak** [comment]
Closing since the associated PR has been merged. Thanks!
